### PR TITLE
Add more storage test

### PR DESCRIPTION
**Pull request by w-yuchen**
Make coverage look a bit better.

### CODE REVIEW

File: src/test/java/seedu/address/storage/JsonAdaptedPropertyTest.java [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+import seedu.address.model.property.PostalCode;
+import seedu.address.model.property.Type;
+import seedu.address.model.remark.Remark;
+
+public class JsonAdaptedPropertyTest {
+    @Test
+    public void validNoClientPropertyCreateTest() throws IllegalValueException {
+        JsonAdaptedProperty adaptedProperty = new JsonAdaptedProperty(WOODLANDS_CRESCENT);
+        assertEquals(adaptedProperty.toModelType(), WOODLANDS_CRESCENT);
+        // assertEquals(adaptedProperty, expectedAdaptedProperty);
```
Comment (w2vgd) [inline src/test/java/seedu/address/storage/JsonAdaptedPropertyTest.java]: What is this commented out testcase for?

File: src/test/java/seedu/address/storage/JsonAdaptedPropertyTest.java [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    }
+
+    @Test
+    public void validWithClientPropertyCreateTest() throws IllegalValueException {
+        JsonAdaptedProperty adaptedProperty = new JsonAdaptedProperty(JURONG);
+        assertEquals(adaptedProperty.toModelType(), JURONG);
+        // assertEquals(adaptedProperty, expectedAdaptedPropertyWithClient);
```
Comment (w2vgd) [inline src/test/java/seedu/address/storage/JsonAdaptedPropertyTest.java]: Similarly for this

File: src/test/java/seedu/address/storage/JsonAdaptedPropertyTest.java [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+        JsonAdaptedProperty adaptedPropertyNoName =
+                new JsonAdaptedProperty(null,
+                        WOODLANDS_CRESCENT.getPropertyType().toString(),
+                        WOODLANDS_CRESCENT.getAddress().toString(),
+                        "",
+                        WOODLANDS_CRESCENT.getPostalCode().toString(),
+                        WOODLANDS_CRESCENT.getDeadline().toString(),
+                        null,
+                        WOODLANDS_CRESCENT.getTags().stream().map(JsonAdaptedTag::new).collect(Collectors.toList()),
+                        null);
```
Comment (w2vgd) [inline src/test/java/seedu/address/storage/JsonAdaptedPropertyTest.java:44]: Perhaps use `PropertyBuilder` here?

**Review by w2vgd — COMMENTED** [review]
Some questions

File: src/test/java/seedu/address/storage/JsonAdaptedPropertyTest.java (hunk 3 above)
Comment (w-yuchen) [reply src/test/java/seedu/address/storage/JsonAdaptedPropertyTest.java:44]: that gives nullpointerexception, so must build with constructor.

**Review by w2vgd — APPROVED** [review]
LGTM

**Pull request merged**